pkg/apis/k0s/v1beta1/cplb.go

```diff
@@ -81,50 +106,148 @@ type VirtualIPs []string
 
 // ValidateVRRPInstances validates existing configuration and sets the default
```

Comment:
Validating changing the state sounds bit bogus. Could we hook defaulting into the config loading "phase" like we do for other parts of the config?